src/models/monster/index.ts

```diff
@@ -48,6 +54,33 @@ const ActionSchema = new Schema<Action>({
   actions: { type: [ActionOptionSchema], index: true },
 });
 
+const ArmorClassSchema = new Schema<ArmorClass>(
+  {
+    _id: false,
+    type: { type: String, index: true, enum: ['dex', 'natural', 'armor', 'spell', 'condition'] },
+    desc: { type: String, index: true },
```

Comment:
Isn't `desc` not on `dex`? It's on every other type, but not on `dex`, so unfortunately I believe you may need to repeat it.

Reply:
I shifted the types so it can be and is optional to make this a little nicer.